components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -194,4 +210,13 @@ export class ProjectListComponent implements OnInit {
   cancelApplyStop() {
     this.closeConfirmApplyStopModal();
   }
+
+  getRulesStatus(project: Project): string {
+    switch (project.status) {
+      case 'NO_RULES': return 'No rules';
+      case 'EDITS_PENDING': return 'Edits pending';
+      case 'RULES_APPLIED': return 'Applied';
+      default: return '';
+    }
```

Comment:
> My preference is for functional-style code, which requires a ternary as given rather than a switch. However, sounds like your preference is strong, so I am happy to oblige.

Oh I prefer functional code, too. But let's not overdo it, nesting ternaries is something I'd rather avoid.  This is more straightforward, thanks for obliging 😉